scripts/install/install-dependencies.sh

```diff
@@ -669,6 +679,11 @@ function test_installed_astrometry_version() {
 # then builds and installs into ${ASTROMETRY_DIR}. Skips as much as
 # possible if able to determine that the version hasn't changed.
 function install_astrometry() {
+  ASTROMETRY_VERSION="${ASTROMETRY_VERSION}" "${THIS_DIR}/install-astrometry.sh"
+  return
+
+
+
```

Comment:
Do you still need everything below if the script is already installing?

Reply:
> IFF we are going to move in the direction of docker

I still don't understand if this question regarding docker is for testing (on travis) or if it applies more generally to an install method. Or even more sorta-generally as an image that could be used, e.g. for compute instance images, travis, etc.

Breaking it up would favor either approach as you could have different files (or I think there are build "flavors" or something) for different targets but build them from the same intermediary stages.